enzy_htp/structure/structure_io/xyz_io.py

```diff
@@ -0,0 +1,42 @@
+"""Generation/construction of Structure objects from PDB files and exporting it vice versa
+Definition of PDB file format (http://www.wwpdb.org/documentation/file-format-content/format33/v3.3.html)
+Also contains util function that requires knoweledge of PDB format
+
+Author: Qianzhen (QZ) Shao, <shaoqz@icloud.com>
```

Comment:
lol change the file head and acknowledge yourself.

Reply:
Whoops haha